styx-client/pom.xml

```diff
@@ -60,7 +60,7 @@
     </dependency>
     <dependency>
       <groupId>org.hamcrest</groupId>
-      <artifactId>hamcrest-all</artifactId>
+      <artifactId>hamcrest</artifactId>
```

Comment:
`hamcrest-all` artifact is deprecated

Reply:
> hamcrest-all.jar: One jar containing all classes of all the other jars. There are no new releases of this library since version 1.3. Please use the single hamcrest.jar instead.